lib/card.rb

```diff
@@ -0,0 +1,10 @@
+class Card
+  attr_reader :question, :answer, :category
+
+
+   def initialize(question, answer, category)
+    @question = question
+    @answer = answer
+    @category = category
+  end
+end
```

Comment:
This looks great! For syntax sake, just do one blank line between the attr_readers and the initialize method.